### PR TITLE
Further usage of thread debug

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by radare** [comment]
did u fixed the warnings that appear in travis?

**Comment by devnexen** [comment]
Found out on Linux _GNU_SOURCE is not enough to enable it, it is also a matter of glibc version. On BSD, headers are enough.

File: libr/core/project.c [hunk 1]
```diff
@@ -354,6 +354,15 @@ R_API RThread *r_core_project_load_bg(RCore *core, const char *prjName, const ch
 	ps->rcPath = strdup (rcPath);
 	RThread *th = r_th_new (projectLoadBackground, ps, false);
 	r_th_start (th, true);
+#if 0
```
Comment (radare) [inline libr/core/project.c]: can we remove this?

Comment (devnexen) [reply libr/core/project.c]: sure ... in fact was not sure it would go without.

**Pull request merged**